calico-enterprise/reference/resources/webhook.mdx

```diff
@@ -0,0 +1,146 @@
+---
+description: API for this Calico Enterprise resource.
+---
+
+# Security Event Webhook
```

Comment:
```suggestion
# Security event webhook
```
Global change: Let's call this a security event webhook, all lowercase.

Reply:
Corrected.